llvm/test/CodeGen/AArch64/selectiondag-dbgvalue-null-crash.ll

```diff
@@ -0,0 +1,45 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
+; RUN: llc < %s -mtriple=aarch64-unknown-linux-gnu | FileCheck %s
+; REQUIRES: system-linux
+; REQUIRES: x86-host
+; REQUIRES: x86-registered-target, aarch64-registered-target
```

Comment:
None of these requires are required. None of this is host dependent, the failure was due to the default target triple